packages/core/src/accordion/accordion-panel.element.scss

```diff
@@ -0,0 +1,85 @@
+@import './../styles/tokens/generated/index';
+@import './../styles/mixins/mixins';
+
+:host {
+  --border-color: #{$cds-token-color-neutral-500};
+  --border-width: #{$cds-token-global-border-width};
+  --color: #{$cds-token-color-neutral-800};
+  --icon-color: #{$cds-token-color-neutral-700};
+  --font-size: #{$cds-token-typography-font-size-4};
+  --font-weight: #{$cds-token-typography-font-weight-regular};
+  --header-background-color: #{$cds-token-color-neutral-50};
+  --header-color: #{$cds-token-color-neutral-800};
+  --header-font-size: #{$cds-token-typography-font-size-3};
+  --header-font-weight: #{$cds-token-typography-font-weight-semibold};
+  --header-open-background-color: #{$cds-token-global-selection-color};
```

Comment:
May not be worth the effort to adjust templates but we could move these to the `cds-accordion-header` element similar to alerts. Then the component doesnt need nested css props on the public API and may not need conditional classes in the component template.

```scss
// cds-accordion-header css
:root {
  --background: #{$cds-token-color-neutral-50};
  --color: #{$cds-token-color-neutral-800};
  --font-size: #{$cds-token-typography-font-size-3};
  --font-weight: #{$cds-token-typography-font-weight-semibold};
}
```

```scss
// cds-accordion-panel css
:host([expanded]) ::slotted(cds-accordion-header) {
  --background: #{$cds-token-global-selection-color};
}
```

```css
cds-accordion-header {
  --background: ...
  --color: ...
  --font-size: ...
  --font-weight: ...
}
```

Reply:
I started out trying to refactor but I think I'll leave it as is for now.

Animation also relies on the conditional css class and some parts of the header template rely on both `expanded` and `disabled` states that are available in the panel, making the refactor a bit less clean-cut than initially anticipated. I'm not opposed to improving things though, so if you think it might still be worth the effort let me know and maybe we can pair on this?

Reply:
Going to resolve since I'm not sure if its really worth the effort especially with the new theming coming soon after.